gcp/modules/gcp-stackdriver-lbm/resources/log_based_metrics/disks_createsnapshot.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Yeah, forgot to mention this in PR description – I found the way to track `compute.disks.createSnapshot` audit events, added metric and condition on that new metric to CouchDB snapshots check policy. This looks more reliable now, since we're tracking actual snapshot creation events and not just records from `k8s-snapshots` logs.